src/package.xml

```diff
@@ -142,6 +142,8 @@
         <members>UTIL_OrgTelemetry_TEST</members>
         <members>UTIL_Profile</members>
         <members>UTIL_Profile_TEST</members>
+        <members>UTIL_SortContacts</members>
```

Comment:
@kdy618 Quick comment here - you'll need to update the package.xml here too since the class names were changed.